Deploy-CAM.ps1

```diff
@@ -3145,8 +3145,6 @@ function Get-CAMRoleDefinition() {
             # Remove ability to access route filters and tables
             'Microsoft.Network/routeFilters/*'
             'Microsoft.Network/routeTables/*'
-            # Remove ability to access secure gateways
```

Comment:
Let's have try and catch on removing the roles.

Reply:
try/catch might get messy since it's a batch thing and we want it to error out if it fails, but i can verify that the role exists if the same thing happens again